src/themes/structural/_generated/Button.css

```diff
@@ -0,0 +1,5 @@
+._button_312p3_1 {
```

Comment:
I ignored generated CSS files and added the `postcss` grunt task to both `dev` and `dist` to make sure it's run before use.